### PR TITLE
[Survey Module] Add warning for when a selected test is not part of the selected visit [Redmine 10430]

### PR DESCRIPTION
**Pull request by davidblader**
https://redmine.cbrain.mcgill.ca/issues/10430

### CODE REVIEW

**Comment by johnsaigle** [comment]
To be discussed in general LORIS meeting.

**Comment by johnsaigle** [comment]
needs leigh. To discuss in a LORIS general meeting (lol)

**Comment by johnsaigle** [comment]
@LeighMac help

**Comment by johnsaigle** [comment]
@davidblader pls respond

**Comment by LeighMac** [comment]
@johnsaigle Sorry, I never check Git... get me on email or carrier pigeon. :)

File: modules/survey_accounts/ajax/TestInVLCheck.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    )
+);
+
+if (!$test_in_VL && $test_has_VL) {
+    $full_name   = $DB->pselectOne(
+        "SELECT Full_name FROM test_names ".
+        "WHERE Test_name=:TN",
+        array("TN" => $test_name)
+    );
+    $warning_msg = "WARNING: $full_name is not normally administered at visit $vl.";
```
Comment (johnsaigle) [inline modules/survey_accounts/ajax/TestInVLCheck.php]: Please wrap this string with `htmlspecialchars`

File: modules/survey_accounts/ajax/TestInVLCheck.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Check if given test is part of given visit label
+ *
+ * PHP Version 5
```
Comment (johnsaigle) [inline modules/survey_accounts/ajax/TestInVLCheck.php]: Change to PHP 7

**Comment by johnsaigle** [comment]
@davidblader A few small changes to make. Was this discussion ever resolved also?

**Comment by davidblader** [comment]
@johnsaigle 
conclusion of various discussions is that this PR is useless as survey instruments are never going to be in test_battery

**Pull request closed without merging**